book/src/developers/designs/replication_coordinator.md

```diff
@@ -133,160 +106,217 @@ kanidmd replication get-certificate
 For each node, a replication configuration is created in json. For A pulling from B.
```

Comment:
^ `kanidmd replication get-certificate` should be formatted to show as a shell command